### ISSUE TITLE
No automatic updated from version 2.3

### ISSUE THREAD

**Issue by TomTurnschuh** [nextcloud/desktop#927]
I manage Nextcloud desktop client installations on several Windows 7 computers.
Even though they were versions 2.3.2 and 2.3.3 (and thus more than a year old), the clients said that there were no updates available.
I now manually updated to the newest version using the install file.

I would assume that many other users cannot automatically update as well.

Is this a bug or are updates from versions 2.3.2 and 2.3.3 intentionally not available yet for everybody?

**Comment by MeCias** [comment]
I had the same issue and updated manually to 2.5.0. Now I noticed that 2.5.1 is available and I get no update information as well and it says" Your installation is at the latest version"

**Comment by GoetheG** [comment]
Had the same issue.

**Comment by GieltjE** [comment]
I've tried to get the update server running (https://github.com/nextcloud/client_updater_server), but no luck.

The only reference on how to install it is that it should be accesible under <nextclouduri>/updates, but it's never accessed.

This should be considered as a serious problem as I've never seen a client be updated or prompted for an update resulting in updateing every client we come across......

**Comment by StefvanEgmond** [comment]
Same issue here with 2.5.1final

**Comment by huste** [comment]
Same here on Win10 64bit: 2.5.1 final installed (build 20181204) and by 2019-08-15, it says there's no update available.

**Comment by stanelie** [comment]
2.5.2 shows "No updates available", while 2.5.3 is available at https://nextcloud.com/install/#install-clients .

**Comment by huste** [comment]
Well, as everyone who noticed this issue obviously upgraded manually by downloading a current build, I'll just bump this issue again. I've seen it a week ago, when I told a friend to check for upgrades (he complained that the nextcloud client told him that his most recent nextcloud server installation was too new): There were none, according to the client. But he had not the most recent but a two year old version installed. Downloaded the current version from the homepage manualle, installed it and was happy.

**Comment by huste** [comment]
I'll as ka question: If I read it correctly (I never did C++ properly nor do I have en IDE or compilers), the URL that is checked for updates is https://updates.nextcloud.org/client/ (as set in NEXTCLOUD.cmake for APPLICATION_UPDATE_URL. OWNCLOUD_UPDATE_URL ist not set anywhere, so APPLICATION_UPDATE_URL ist used. Right?
When I wget this URL, all I get is an empty response.

So the question is: Which URL shall be checked? Is any developer or nextcloud-overview-person here who can answer and fix this?

**Comment by misch7** [comment]
We've recently discovered the error was at the update server.

Since we're going to release version 2.6 by the end of next week we decided to wait till then for a change. Otherwise you would get the updates to intermediate versions and just would have to update all-over-again to the latest version. 😼

**Comment by huste** [comment]
Thanks for the update, Michael.

**Comment by vincent1890** [comment]
> We've recently discovered the error was at the update server.
> 
> Since we're going to release version 2.6 by the end of next week we decided to wait till then for a change. Otherwise you would get the updates to intermediate versions and just would have to update all-over-again to the latest version.

Version 2.3.3 (build 1) shows "No updates available", while 2.6.0 is available at https://nextcloud.com/install/#install-clients .

**Comment by vincent1890** [comment]
Does anyone have the same problem with the latest version?

**Comment by misch7** [comment]
Update servers will announce the update by the end of this week. We wanted to be sure, 2.6 is stable enough before we push it out at large scale ;-)

**Comment by vincent1890** [comment]
Actually updated tonight, it works fine thanks for this correction.

**Comment by huste** [comment]
I also just received the update notification. nextcloud.com seems to be a bit overloaded at the moment.

**Comment by misch7** [comment]
Cool, thanks for the feedback! 👍 

I've updated https://github.com/nextcloud/client_updater_server last night (PR: https://github.com/nextcloud/client_updater_server/pull/27) in case you're interested in the technical details ;-)

As you already guessed the client uses APPLICATION_UPDATE_URL and the client_updater_server runs there on our update server and replies with the update URL when some required fields are set in the request (else you get an empty reply).

Now I'm also working on the Mac part, this requires some more work.

Since this issue has been solved, I'm happily closing it 😸
